### ISSUE TITLE
[bug] TypeError: Cannot read properties of undefined (reading 'readResources')

### ISSUE THREAD

**Issue by juniorforlife** [inlang/inlang#1160]
### Problem

I just installed @inlang/sdk-js for my SvelteKit project and immediately I saw    
1. dev server takes a lot longer to start.
2. in the terminal there are a bunch of `files` string being logged
3. error: `TypeError: Cannot read properties of undefined (reading 'readResources')
     at Module.reloadResources (/Users/mike/Documents/project/node_modules/.pnpm/@inlang+sdk-js@0.9.2_@sveltejs+kit@1.22.3_esbuild@0.18.14_svelte@4.0.5/node_modules/@inlang/sdk-js/dist/adapter-sveltekit/runtime/server/state.js:36:64)`
![image](https://github.com/inlang/inlang/assets/68360917/da2228d6-7a52-4ce3-95f0-036ca3683823)

### Expected behavior

No performance issue, no weird logs and error

### Reproduction

I followed this tutorial by Ivan Hofer https://www.youtube.com/watch?v=Y6IbPfMU1xM

### `inlang.config.js` file

```JavaScript
/**
 * @type { import("@inlang/core/config").DefineConfig }
 */
export async function defineConfig(env) {
	const { default: jsonPlugin } = await env.$import(
		"https://cdn.jsdelivr.net/npm/@inlang/plugin-json@3/dist/index.js"
	)
	const { default: sdkPlugin } = await env.$import(
		"https://cdn.jsdelivr.net/npm/@inlang/sdk-js-plugin@0.9.2/dist/index.js"
	)

	return {
		referenceLanguage: "en",
		plugins: [
			jsonPlugin({
				pathPattern: "./languages/{language}.json",
			}),
			sdkPlugin({
				languageNegotiation: {
					strategies: [{ type: "localStorage" }]
				}
			}),
		],
	}
}
```

**Comment by ivanhofer** [comment]
HI @juniorforlife, please create a repoistory where we can reproduce the issue.

**Comment by ivanhofer** [comment]
@NilsJacobsen please release a new version of the JSON plugin. The console log comes from there..
https://cdn.jsdelivr.net/npm/@inlang/plugin-json@3/dist/index.js

**Comment by ivanhofer** [comment]
> 1. dev server takes a lot longer to start

How much longer? Can you provide more information?

**Comment by juniorforlife** [comment]
@ivanhofer it's weird that when I tried to setup a new repo the `TypeError` only happened once then I couldn't reproduce it. It happened consistently when I first tried inlang. I'll try to reproduce it

**Comment by NilsJacobsen** [comment]
The "files" log should be removed in https://github.com/inlang/inlang/commit/de0fe4e39df8e7b64d8b29d34ad19b7533df9823 5 days ago. I purged jsdelivr now. It should not be there anymore.

**Comment by ivanhofer** [comment]
This should be fixed in version `0.10.1`
